Extension/src/common.ts

```diff
@@ -785,6 +787,7 @@ export function extractCompilerPathAndArgs(inputCompilerPath: string): CompilerP
                 additionalArgs = compilerPath.substr(endQuote + 1).split(" ");
                 additionalArgs = additionalArgs.filter((arg: string) => { return arg.trim().length !== 0; }); // Remove empty args.
                 compilerPath = compilerPath.substr(1, endQuote - 1);
+                compilerName = compilerPath.replace(/^.*(\\|\/|\:)/, '');
```

Comment:
Does this work when a path is specificied as an arg to the compiler path?

Reply:
I moved this line after trying to get a potentialCompilerPath. It now only gets the compiler if a valid compilerPath was found, meaning that is when compilerPath only contains the compiler path and it doesn't have any args. So `compilerPath.replace(/^.*(\\|\/|\:)/, '');` will only be applied to a compiler path without args.